include/qpxsettings.hpp

```diff
@@ -0,0 +1,190 @@
+/*
+ * This file is part of the QPxTool project.
+ * Copyright (C) 2008-2010 Gennady "ShultZ" Kozlov <qpxtool@mail.ru>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file "COPYING" for the exact licensing terms.
+ *
+ */
+
+#ifndef _QPXSETTINGS_H
+#define _QPXSETTINGS_H
+
+#include <QColor>
+#include <QRect>
+#include <QString>
+#include <QStringList>
+
+#include <inttypes.h>
+
+#include "db_connection.hpp"
+
+struct colorSet {
+  QString name;
+  QColor bg, grid, rspeed, wspeed, errc[8], jitter, asymm, fe, te, tapit,
+      taland;
+};
+
+static const colorSet defColors = {"Default",
+                                   0xFFFFFF,
+                                   0x808080,
+                                   0x009000,
+                                   0xC00000,
+                                   {0x009000, 0x00C000, 0x00FF00, 0xFFAA00,
+                                    0x0055FF, 0x550080, 0xFF0000, Qt::black},
+                                   0x00A000,
+                                   0xA00000,
+                                   0xA00000,
+                                   0x0000A0,
+                                   0xA00000,
+                                   0x0000A0};
+
+class Scale {
+public:
+  Scale(QString iname);
```

Comment:
Codacy found an issue: [Class 'Scale' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)